pkg/debug/scripts/collect-info.sh

```diff
@@ -44,9 +44,9 @@ DIR="$TMP_DIR/$INFO_DIR"
 mkdir -p "$DIR"
 mkdir -p "$DIR/network"
 
-# Install GNU version of the 'ps', 'tar' and other tools
+# Install GNU version of the 'ps' and other tools
 echo "- install GNU tools"
-apk add procps tar dmidecode >/dev/null 2>&1
+apk add procps dmidecode >/dev/null 2>&1
```

Comment:
If the network is flaky or disconnected, doesn't leaving the apk add mean that that the script will sit there for a few minutes waiting for the apk add to time out?

Reply:
Added check for internet access :
https://github.com/lf-edge/eve/pull/3388/commits/4c06ad4a6d92fbd7367fcdb22f5fe4aa3dd52133